src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -15,93 +15,140 @@
 package com.google.sps.servlets.utils;
 
 import java.math.BigDecimal;
+import java.util.ArrayList;
 import java.util.Comparator;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.stream.Collectors;
+import java.util.PriorityQueue;
 
 public class Statistics {
-  private static final double LOWER_END_VAL = -1.0;
-  private static final double UPPER_END_VAL = 1.0;
-  private static final BigDecimal INTERVAL = BigDecimal.valueOf(0.2);
-  private static final BigDecimal UPPER_END = BigDecimal.valueOf(UPPER_END_VAL);
-  private static final BigDecimal LOWER_END = BigDecimal.valueOf(LOWER_END_VAL);
+  private static final double LOWER_SCORE_VAL = -1.0;
+  private static final double UPPER_SCORE_VAL = 1.0;
+  private static final double SCORE_INTERVAL_VAL = 0.2;
+  // TODO: set the current Top n comment as 1; will be updated once we have the input from front-end
```

Comment:
nit: Reverse the order here - Update the number of top comments stored to be user defined. Right now we are hard-coding it as 1.

Reply:
Problem solved with overloading statistics object:) No need to mention it here